pkg/virt-launcher/monitor_test.go

```diff
@@ -120,6 +121,8 @@ var _ = Describe("VirtLauncher", func() {
 			syscall.Kill(pid, syscall.SIGTERM)
 		}
 		gracefulShutdownCallback := func() {
+			gracefulShutdownCallbackTriggeredLock.Lock()
```

Comment:
Could we use a channel here too?

Reply:
Do you mean replacing the boolean `gracefulShutdownCallbackTriggered` with a channel?
How would I deal with `Expect(gracefulShutdownCallbackTriggered).To(BeFalse())`?

Reply:
Something like

```
Eventually(channel).Should(BeClosed())
```
Should work

Reply:
Right, that would replace the `BeTrue()` test, but what about `BeFalse()`? Just skip it?

Reply:
Did not try this but I would assume `Consistenlty(channel).Should(Not(BeClosed)))`

Reply:
Interesting, I wonder how that's checked without doing a read(), probably some golang magic. I'll try that, thank you!

Reply:
Seems to work just fine, force-pushed.